standards/out-of-hours.md

```diff
@@ -0,0 +1,62 @@
+
+## Assessing your team's ability to provide out of hours support
+
+To run a sustainable out of hours rota, you must first ensure that either:
+
+- You have enough staff (including contractors) to support the service out of hours sustainably (realistically this means at least six people, ideally eight), and have the ability to pay them for their time supporting the service
+
+Or:
+
+- You have budget to pay a third party to provide your support sustainably, and have the ability to train them sufficiently before they are expected to start supporting it.
+
+If you are unable to do either of these, you cannot support your service out of hours without risking people's health and well-being. If that is the case, you should escalate this risk and not provide support until you are in a position to do one or other of those.
+
+## Managing a sustainable out of hours rota
+
+Being on call can be stressful, and can impact people's mental health and personal lives if not managed well. If that happens, people will withdraw from the rota, leaving the service unsupported.
+
+To avoid that, a few core criteria must be met:
+
+- No one person is on call 50% of the time or more, and ideally no more than 30% of the time. This, along with the next criteria, mean that you will need at least six people, and ideally eight, to run a rota sustainably.
+- At least two people are on call at any one time, and know which of them will be contacted first (primary on-call) and second (secondary on-call).
+- If your service regularly has problems out of hours, or problems typically take multiple hours to resolve, you will need to take that into account when designing your rota. These sorts of situations will require more people, shorter shifts, or more layers of support.
+- If your service regularly has problems out of hours, your incident retrospectives should be ensuring that appropriate learning and investment is happening to improve how often the service has problems.
+- The rota repeats regularly, so people know when they are on call over the coming weeks and months, and can plan accordingly.
+- People on call are appropriately paid for being available on call and for responding to incidents.
+- People on call know when they are expected to be available when on call, whether 24/7, 7am-10pm, or some other, agreed hours.
+- People on call can trade shifts with one another when they need to.
+- People on call for the first time are on call with someone experienced to being on call on their support rota, and ideally someone they know well.
```

Comment:
time are on call with someone experienced to being on call on their support rota -> time are scheduled to be on their rota with someone very experienced